data/interface.go

```diff
@@ -177,3 +177,21 @@ type SyncStatisticsHandler interface {
 	NumMissing() int
 	IsInterfaceNil() bool
 }
+
+// TransactionWithFeeHandler represents a transaction structure that has economics variables defined
+type TransactionWithFeeHandler interface {
+	GetGasLimit() uint64
+	GetGasPrice() uint64
+	GetData() []byte
+	GetRcvAddr() []byte
+	GetValue() *big.Int
+}
+
+// UserAccountHandler models a user account
+type UserAccountHandler interface {
+	RetrieveValueFromDataTrieTracker(key []byte) ([]byte, error)
```

Comment:
I think we could have a shorter name -GetValue - GetKeyValue stuff liek that. There is no need to now that it is under DataTrieTracker

Reply:
This method is only used in the elastic-indexer-go repository. I use this name because is more explicit ( this method call method  `RetriveValue` from `Data Trie Tracker `). I would like to keep this name.

Reply:
we have something similar in case of arwen - but we made it much simpler :)

Reply:
Will take a look and made a task